components/organisms/SuggestedRepoList/suggested-repo-list.tsx

```diff
@@ -15,7 +15,7 @@ const SuggestedRepositoriesList = ({ reposData, onAddRepo }: SuggestedRepositori
         Suggested Repositories:
       </Title>
 
-      <div className="flex flex-col gap-3 pb-10 mt-6 border-b">
+      <div className="flex flex-col gap-3 mt-6">
```

Comment:
```suggestion
      <div className="flex flex-col gap-3 pb-10 mt-6 border-b">
```

I think we should go with another approach.  [here](https://github.com/open-sauced/insights/blob/298718f1e7c5bd28df93e2e8e612e25a8c6dd097/components/organisms/Footer/footer.tsx#L63C1-L63C1) we should remove border-top. @bdougie  it is good like that 
![image](https://github.com/open-sauced/insights/assets/37635083/d4ce395a-dbc4-47cc-afdf-e3aeeb8cc9b7)